File: Course_1-Algorithmic_Toolbox/Week-3/Excercise_Challenges/1_money_change/change.cpp
#include <iostream>
using namespace std;

int get_change(int m)
{
    int coin[] = {1, 5, 10};
    int con = 0;

    for(int i = 2; i >= 0; --i){
        while(coin[i] <= m){
            con++;
            m -= coin[i];   
        }
    }
    return con;
}

int main(void)
{
    int m;
    cin >> m;
    cout << get_change(m) << '\n';

    return 0;
}

File: Course_1-Algorithmic_Toolbox/Week-3/Excercise_Challenges/1_money_change/change.py
def get_change(m):
    coin = [1, 5, 10]
    con = 0

    for c in coin[::-1]:
        while c <= m:
            con += 1
            m -= c

    return con


if __name__ == '__main__':
    m = int(input())
    print(get_change(m))
